loopchain/channel/channel_service.py

```diff
@@ -283,13 +278,16 @@ def _get_node_type_by_peer_list(self):
                 }
                 self.__peer_manager.add_peer(peer_info)
             else:
-                self._load_peers_from_iiss()
+                self.__peer_manager.load_peers_from_iiss()
+
+            if not conf.LOAD_PEERS_FROM_IISS:
+                await self.__peer_manager.load_peers_from_file()
```

Comment:
I'm not sure that I catched your intention correctly, but `PeerManager.load_peers_from_file` may located at: 
https://github.com/icon-project/loopchain/blob/7f3389a24c632c0afc798f7e312ee49ec5f9ad8f/loopchain/baseservice/peer_manager.py#L163-L164

Reply:
There are separated two options for only use registration feature in IISS.